src/quo2/components/inputs/locked_input/style.cljs

```diff
@@ -0,0 +1,22 @@
+(ns quo2.components.inputs.locked-input.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn info-box-container
+  [theme]
+  {:flex-direction     :row
+   :border-radius      12
+   :align-items        :center
+   :background-color   (colors/theme-colors colors/neutral-10
+                                            colors/neutral-80-opa-80 theme)
+   :width              :100%
```

Comment:
We don't recommend using `%` values for `width`. In most cases, we can achieve the same result using flexbox properties. In this case, I think we can remove width and will have the same result.